### PR TITLE
Document Debian package

### LINKED ISSUE #701: Document the Debain Package
icalendar has a debian package:

```shell
sudo apt-get install python-icalendar
```

To close the issue:
- [ ] link to the package website e.g. https://tracker.debian.org/pkg/python-icalendar 
- [ ] add the installation method for Debian/Ubuntu

File to change: [install.html](https://icalendar.readthedocs.io/en/latest/install.html)

See also:
- https://github.com/niccokunzmann/python-recurring-ical-events/pull/162/files

### PR DESCRIPTION
**Pull request by niccokunzmann**
This fixes #701

----
📚 Documentation preview 📚: https://icalendar--718.org.readthedocs.build/

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Changelog
 
 Minor changes:
 
-- ...
+- Document the Debian package, see `Issue 701 <https://github.com/collective/icalendar/issues/701>`_
```
Comment (stevepiercy) [inline CHANGES.rst]:
```suggestion
- Document the Debian package. See `Issue 701 <https://github.com/collective/icalendar/issues/701>`_.
```

File: docs/install.rst [hunk 2]
```diff
@@ -14,6 +19,16 @@ package, like this:
 
     >>> import icalendar
 
+Debian/Ubuntu
```
Comment (stevepiercy) [inline docs/install.rst]:
```suggestion
Debian or Ubuntu
```

File: docs/install.rst [hunk 3]
```diff
@@ -14,6 +19,16 @@ package, like this:
...
+-------------
```
Comment (stevepiercy) [inline docs/install.rst]:
```suggestion
----------------
```

File: docs/install.rst [hunk 4]
```diff
@@ -14,6 +19,16 @@ package, like this:
...
+
+You can install the `python-icalendar package <https://tracker.debian.org/pkg/python-icalendar>`_
+for Debian and derivatives.
```
Comment (stevepiercy) [inline docs/install.rst]:
```suggestion
for Debian or its derivatives.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor touch-up

**Comment by niccokunzmann** [comment]
Thanks for the documentation review! I assume it is ok to merge when the tests pass...

**Pull request merged**